scripts/exglobal_efsoi_fv3gfs.sh.ecf

```diff
@@ -0,0 +1,520 @@
+#!/bin/ksh
+################################################################################
+####  UNIX Script Documentation Block
+#                      .                                             .
+# Script name:         exglobal_efsoi_fv3gfs.sh.ecf
+# Modified from:         exglobal_enkf_update_fv3gfs.sh.ecf
+# Script description:  Make global_enkf update
+#
+# Author:        Rahul Mahajan      Org: NCEP/EMC     Date: 2017-03-02
+#
+# Abstract: This script runs the global_enkf update
```

Comment:
If it does this, then how is it different from `exglobal_enkf_update.sh`

Reply:
Thank you for all the above comments.  I have updated exglobal_efsoi_fv3gfs.sh.ecf according.